### PR TITLE
Modified test cases and examples of MySql for fix issue.

### PR DESCRIPTION
**Pull request by LucasYao93**
## Description

Modified test cases and examples of MySql.

## Includes

Change New-AzMySqlServerReplica name and “replica” instead of name
#11939
#11938

*-AzMySqlFirewallRule Use only one parameter when only one IP
needs to be authorized
#11933

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/MySql/docs/Update-AzMySqlFirewallRule.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ rule 0.0.0.2        0.0.0.3
 
 This cmdlet updates MySql Firewall Rule by name.
 
-### Example 2: Update MySql Firewall Rule by identity.
+### Example 2: Update MySql Firewall Rule by identity
```
Comment (msJinLei) [inline src/MySql/docs/Update-AzMySqlFirewallRule.md:44]: Unnecessary change

File: src/MySql/docs/Update-AzMySqlFirewallRule.md [hunk 2]
```diff
@@ -51,7 +51,19 @@ Name StartIPAddress EndIPAddress
 rule 0.0.0.2        0.0.0.3
 ```
 
-These cmdlets update MySql Firewall Rule by identity.
+These cmdlets update MySql Firewall Rule by identity .
```
Comment (msJinLei) [inline src/MySql/docs/Update-AzMySqlFirewallRule.md:54]: redundant space before '.'

File: src/MySql/examples/Update-AzMySqlFirewallRule.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ rule 0.0.0.2        0.0.0.3
 
 This cmdlet updates MySql Firewall Rule by name.
 
-### Example 2: Update MySql Firewall Rule by identity.
+### Example 2: Update MySql Firewall Rule by identity
```
Comment (msJinLei) [inline src/MySql/examples/Update-AzMySqlFirewallRule.md:12]: Unnecessary change

File: src/MySql/examples/Update-AzMySqlFirewallRule.md [hunk 4]
```diff
@@ -19,4 +19,16 @@ Name StartIPAddress EndIPAddress
 rule 0.0.0.2        0.0.0.3
 ```
 
-These cmdlets update MySql Firewall Rule by identity.
\ No newline at end of file
+These cmdlets update MySql Firewall Rule by identity .
```
Comment (msJinLei) [inline src/MySql/examples/Update-AzMySqlFirewallRule.md:22]: redundant space before '.'

File: src/MySql/docs/Update-AzMySqlFirewallRule.md (hunk 1 above)
Comment (LucasYao93) [reply src/MySql/docs/Update-AzMySqlFirewallRule.md]: Done

File: src/MySql/examples/Update-AzMySqlFirewallRule.md (hunk 3 above)
Comment (LucasYao93) [reply src/MySql/examples/Update-AzMySqlFirewallRule.md]: Done

File: src/MySql/docs/Update-AzMySqlFirewallRule.md (hunk 2 above)
Comment (LucasYao93) [reply src/MySql/docs/Update-AzMySqlFirewallRule.md]: Done

File: src/MySql/examples/Update-AzMySqlFirewallRule.md (hunk 4 above)
Comment (LucasYao93) [reply src/MySql/examples/Update-AzMySqlFirewallRule.md]: Done
